devops/bin/update_server_list.bash

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+source /Users/bwu2/.github/token.sh
+
+path_shard0="https://raw.githubusercontent.com/harmony-one/nodedb/master/mainnet/shard0.txt"
+path_shard1="https://raw.githubusercontent.com/harmony-one/nodedb/master/mainnet/shard1.txt"
+path_shard2="https://raw.githubusercontent.com/harmony-one/nodedb/master/mainnet/shard2.txt"
+path_shard3="https://raw.githubusercontent.com/harmony-one/nodedb/master/mainnet/shard3.txt"
+
+# PLEASE update the path to locate the key
+path_keys=/Users/andywu/aws/keys
+
+
+declare -a ip_array0
+declare -a ip_array1
+declare -a ip_array2
+declare -a ip_array3
+
+# retrieve host file
+download_hosts_file () {
+
+	# download list of IP for shard 0
+	curl -H "Authorization: token $git_token" \
```

Comment:
use a for loop

Reply:
wrote the script in a rush. yes, should be done in a for a loop.